test/e2e/flowaggregator_test.go

```diff
@@ -264,20 +281,71 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						}
 					}
 				}
-				if checkNetworkPolicy {
+				if checkK8sNetworkPolicy {
 					// Check if records have both ingress and egress network policies.
 					if !strings.Contains(record, ingressNetworkPolicyName) {
 						t.Errorf("Record does not have NetworkPolicy name with ingress rule")
 					}
 					if !strings.Contains(record, fmt.Sprintf("%s: %s", "ingressNetworkPolicyNamespace", testNamespace)) {
 						t.Errorf("Record does not have correct ingressNetworkPolicyNamespace")
 					}
+					if !strings.Contains(record, "ingressNetworkPolicyUID") {
```

Comment:
For K8s network policy, the UID can be customized. But for Antrea network policy, the `AntreaNetworkPolicySpecBuilder` doesn't support us to customize the UID. Do you think that's ok for us to only validate K8s np's UID? I feel testing on K8s np is indeed good enough, but it may looks a bit inconsistent, for testing on the different fields with different flow types?

Reply:
Testing only for k8s NP is fine with me.